### PR TITLE
created slide and some corrections in actual doc

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

**Comment by surajssd** [comment]
@LalatenduMohanty Finally I was able to figure out how to use asciidoctor and revealjs.

Bunch of observations:
- In doc in some places the section is so huge to fit on page, but when some changes are made to divide that section, the doc numbering becomes weird. For e.g. https://github.com/container-workbook/docker-workshop/blob/6e0b0c2682a47ad9128175ea8dbf835d30c1e4a8/docker_basic_workshop.adoc#73-kernel-security-features
- So what do we give more priority to the numbering in doc or the way slides look (Is it okay if slides have overflowing content?)

screenshot from 2016-09-20 17-55-11

File: docker_basic_workshop.adoc [hunk 1]
```diff
@@ -257,12 +272,13 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
...
-* Modify a file
 ----------
 $ docker run -i -t fedora /bin/bash
-[root@88226d270d64 /]# 
-[root@88226d270d64 /]# echo "FUDCon workshop 2015" > /etc/motd
+----------
 
+* Modify a file
+----------
+[root@88226d270d64 /]# echo "FUDCon workshop 2015" > /etc/motd
```
Comment (LalatenduMohanty) [inline docker_basic_workshop.adoc:281]: @surajssd lets make it a generic string.

**Comment by LalatenduMohanty** [comment]
@surajssd I do not think we need to check-in the html file.

**Comment by LalatenduMohanty** [comment]
> So what do we give more priority to the numbering in doc or the way slides look (Is it okay if slides have overflowing content?)

The initial sections i.e. the theory part before actual docker commands should not have overflowing content. For other sections, it is ok.

**Comment by surajssd** [comment]
@LalatenduMohanty since I have divided the doc in new PR so most of the problems will be solved!

**Comment by LalatenduMohanty** [comment]
@surajssd please rebase.

**Pull request closed without merging**